tools/cli/templates/projects/webpack/README.md

```diff
@@ -0,0 +1,36 @@
+# The default coherent-guic-cli Webpack template
+
+This is the default template for [GameUIcomponents](https://coherentlabs.github.io/GameUIComponents/en/) using [Vite](https://webpack.js.org/) creates using [coherent-guic-cli](https://www.npmjs.com/package/coherent-guic-cli).
+
+Uses Webpack with various plugins and loaders to setup an environment that can handle
+- importing JavaScript modules - [babel-loader](https://webpack.js.org/loaders/babel-loader/)
+- importing CSS - [css-loader](https://webpack.js.org/loaders/css-loader/) and [style-loader](https://webpack.js.org/loaders/style-loader/)
+- loading HTML and injecting dependencies using link and script tags for styles and JS - [html-webpack-plugin](https://webpack.js.org/plugins/html-webpack-plugin/)
+- loading assets such as images - [file-loader](https://v4.webpack.js.org/loaders/file-loader/)
+
+## How to Run
+
+Run
+```
+coherent-guic-cli create webpack <directory>
+```
+
+After the project has been created and its dependencies installed, run:
+```
+cd <directory>
+```
+
+and use one of the following commands:

```

Comment:
Same as before

Reply:
Yes, but if you're just checking out the template and you decide you want to create a project from it? :D - https://github.com/CoherentLabs/GameUIComponents/tree/feature/enhance-cli/tools/cli/templates/projects/vite